DependencyInjection/SmartboxIntegrationFrameworkExtension.php

```diff
@@ -301,7 +301,6 @@ protected function loadQueueDrivers(ContainerBuilder $container)
 
                     $driverDef = $container->findDefinition('smartesb.drivers.queue.phpamqplib');
                     $driverDef->addMethodCall('setId', [$driverId]);
-//                    $driverDef->addMethodCall('configure', [$driverConfig['host'], null, null, $driverConfig['format']]);
```

Comment:
This line should maintained here because from the driver yaml config the user can setup the format the of the data the driver is going to read and write to the queue

Reply:
Ok, it was modified.